
class intro {
    public static void main(String[] args) {
           System.out.println("1");
           System.out.println("2");
           System.out.println("3");
           System.out.println("4");
        //    Exception in thread "main" java.lang.ArithmeticException: / by zero
        // it break flow of program due to  one error , soln  is use try and catch - it cannot pause program
           System.out.println(100/0);
           System.out.println("5");
           System.out.println("6");
           System.out.println("7");

    }
}